src/OSFramework/DataGrid/Interface/IDataSource.ts

```diff
@@ -1,5 +1,5 @@
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-namespace OSFramework.Grid {
+namespace OSFramework.DataGrid.Grid {
```

Comment:
`OSFramework.DataGrid.Grid.IGrid` is starting to sound a bit weird @glundgren93 @rugoncalves @joselrio

Reply:
what do you suggest?

Reply:
I don't know the whole architecture to provide a good hint. But reading the command it seems confusing out of context and maybe I expected something closer to `OSFramework.DataGrid.IGrid`, but probably due to our architecture, it doesn't make sense. This is not a blocker, just an opening for discussion.
@rugoncalves @joselrio @JoaoFerreira-FrontEnd any insight?

Reply:
We've a similar approach at other repos, and I also can't suggest a different and best approach :(